src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2360,8 +2360,8 @@ void SessionImpl::decreaseTorrentsQueuePos(const QVector<TorrentID> &ids)
         torrentQueue.pop();
     }
 
-    for (auto i = m_downloadedMetadata.cbegin(); i != m_downloadedMetadata.cend(); ++i)
-        torrentQueuePositionBottom(m_nativeSession->find_torrent(*i));
+    for (const lt::torrent_handle &torrentHandle : m_downloadedMetadata)
```

Comment:
Seems you need `asConst()`
```suggestion
    for (const lt::torrent_handle &torrentHandle : asConst(m_downloadedMetadata))
```
and at other places too.

Reply:
All done.